README.md

```diff
@@ -404,15 +404,97 @@ Note that named function signatures in an interface block indicate methods, rath
 
 ```js
 interface Collection {
-  (signatureParam: Any) => Any, // Collection() signature
-  method1(items: [...Array]) => [...Array], // method
+  (signatureParam: Any) => Any,              // Collection() signature
+  method1(items: [...Array]) => [...Array],  // method
   method2(items: [...Object]) => [...Object] // method
 }
+```
+
+### Key Definition
+
+For convenience, interfaces that feature similar keys can use property templates.
+These templates let us define labels—and optionally types—of objects' keys.  
+*Internally* there are only 2 valid types of keys: `String` and `Symbol`.
+In this regard, [regular expression literal](#regular-expression) and [`Number`](#number) will be automatically mapped to `String`. If ommited the type defaults to `String`.
+
+```js
+{
+  [id1]: {
+    skating: {time: 1000, money: 300},
+    'cooking': {time: 9999, money: 999}
+  },
+  [id2]: {
+    "jogging": {time: 300, money: 0}
+  }
+  etc.
+}
+```
+
+The preceding object can be expressed using this interface:
+
+```js
+interface clientHobbies {
+  [id: Symbol]: {
+    [hobby]: {          // [hobby: String]: {
+      time: Number,
+      money: Number
+    }
+  }
+}
+```
+
+You can even go further to improve the semantics of your interfaces:
+
+```js
+interface Expenditure {
+  time: Number,
+  money: Number
+}
+
+interface clientHobbies {
+  [id: Symbol]: {
+    [hobby]: Expenditure
+  }
+}
+```
+
+#### Number
+
+Types that are automatically coerced into a string can also be documented:
```

Comment:
Okay…

Sorry but what's your point then? I am stating exactly what you just said.
There's also the introduction which is pretty clear:
> *Internally* there are only 2 valid types of keys: `String` and `Symbol`.
In this regard, [regular expression literal](#regular-expression) and [`Number`](#number) will be automatically mapped to `String`.


I am gonna try to guess: are you implying that you don't see how convenient it would be for the user? Would you prefer a more explicit approach? e.g. `String(Number)` cf  https://github.com/ericelliott/rtype/pull/115/commits/a014982b48cca60eff80dc8509eddf5c38e89a87#diff-04c6e90faac2675aa89e2176d2eec7d8R480